lab/changelog.md

```diff
@@ -4,6 +4,31 @@ title: Changelog
 sidebar_label: Changelog
 ---
 
+## v2.4.0 - Dec 2, 2022
+
+## What's New
+
+* Support for manual transaction workflow has been enabled using transaction commands `BEGIN`, `COMMIT`, and `ROLLBACK`.
+* Cypher intellisense has been updated to suggest new Cypher features from Memgraph 2.4.0 such as:
+  * Privileges for user-role authorization.
+  * Commands and privileges for label-based authorization.
+  * Manual transaction commands: `BEGIN`, `COMMIT`, `ROLLBACK`.
+  * Checking configuration with `SHOW CONFIG`.
+  * All shortest path algorithm `allShortest`.
+  * Graph projection function `project`.
+  * Additional query module signature that accepts a projected graph as an optional first argument.
+* Graph results view will check for nodes and relationships in arrays and projected graphs. It simplifies
+  the visualization of a projected graph or an array of nodes/relationships without using `UNWIND`.
+
+## Bug Fixes
+
+* Once the table results view is selected, an automatic switch to the graph view won't happen on the next query run.
```

Comment:
```suggestion
* Once the table results view is selected, the results of the following query run will also preview in the table results view, instead of automatically switching to the graph view.
```